content/zh-cn/docs/reference/command-line-tools-reference/kubelet.md

```diff
@@ -2203,26 +2188,25 @@ Optional absolute name of cgroups to create and run the runtime in.
 <tr>
 <td></td><td style="line-height: 130%; word-wrap: break-word;">
 <!--
-Timeout of all runtime requests except long running request - <code>pull</code>, <code>logs</code>, <code>exec</code> and <code>attach</code>. When timeout exceeded, kubelet will cancel the request, throw out an error and retry later. (DEPRECATED: This parameter should be set via the config file specified by the Kubelet's <code>--config</code> flag. See https://kubernetes.io/docs/tasks/administer-cluster/kubelet-config-file/ for more information.)
+Timeout of all runtime requests except long running request - <code>pull</code>, <code>logs</code>, <code>exec</code> and <code>attach</code>. When timeout exceeded, kubelet will cancel the request, throw out an error and retry later. (DEPRECATED: This parameter should be set via the config file specified by the Kubelet's <code>--config</code> flag. See <a href="https://kubernetes.io/docs/tasks/administer-cluster/kubelet-config-file/">kubelet-config-file</a> for more information.)
 -->
 设置除了长时间运行的请求（包括 <code>pull</code>、<code>logs</code>、<code>exec</code>
 和 <code>attach</code> 等操作）之外的其他运行时请求的超时时间。
 到达超时时间时，请求会被取消，抛出一个错误并会等待重试。
-已弃用：应在 <code>--config</code> 所给的配置文件中进行设置。
-（<a href="https://kubernetes.io/zh/docs/tasks/administer-cluster/kubelet-config-file/">进一步了解</a>）
+（已弃用：应在 <code>--config</code> 所给的配置文件中进行设置。
+请参阅 <a href="https://kubernetes.io/docs/tasks/administer-cluster/kubelet-config-file/">kubelet-config-file</a> 了解更多信息。）
 </td>
 </tr>
 
 <tr>
-<td colspan="2">--seccomp-profile-root string&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;<!--Default: <code>/var/lib/kubelet/seccomp</code>-->默认值：<code>/var/lib/kubelet/seccomp</code></td>
+<td colspan="2">--seccomp-default RuntimeDefault</td>
 </tr>
 <tr>
 <td></td><td style="line-height: 130%; word-wrap: break-word;">
-<!--
-&lt;Warning: Alpha feature&gt; Directory path for seccomp profiles. (DEPRECATED: will be removed in 1.24 or later, in favor of using the <code><root-dir>/seccomp</code> directory)
+<!-- 
+&lt;Warning: Alpha feature&gt; Enable the use of <code>RuntimeDefault</code> as the default seccomp profile for all workloads. The <code>SeccompDefault</code> feature gate must be enabled to allow this flag, which is disabled by default.
 -->
-&lt;警告：alpha 特性&gt; seccomp 配置文件目录。
-已弃用：将在 1.23 或更高版本中移除，以使用 <code>&lt;root-dir&gt;/seccomp</code> 目录。
+&lt;警告：alpha 特性&gt; 启用 <code>RuntimeDefault</code> 作为所有工作负载的默认 seccomp 配置文件。 <code>SeccompDefault</code> 特性门控必须启用以允许此标志，默认情况下禁用。
```

Comment:
Maybe it can be split into 2 lines instead.
```suggestion
&lt;警告：alpha 特性&gt; 启用 <code>RuntimeDefault</code> 作为所有工作负载的默认 seccomp 配置文件。<code>SeccompDefault</code> 特性门控必须启用以允许此标志，默认情况下禁用。
```